src/index.html

```diff
@@ -9,11 +9,19 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <title>Search-bar Airbnb</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+<form action="" class="searchbar-1" data-qa="big">
```

Comment:
add attribute `method`. It's mandatory for forms
```suggestion
<form action="#" class="searchbar-1" data-qa="big">
```